hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/DroolsRulesEngineImpl.java

```diff
@@ -50,6 +50,8 @@
     // private final MsgLogger msgLog = MsgLogger.LOGGER;
     private final Logger log = Logger.getLogger(DroolsRulesEngineImpl.class);
     private static final String SESSION_NAME = "hawkular-alerts-engine-session";
+    private static final long PERF_FILTERING_THRESHOLD= 3000L; // 3 seconds
```

Comment:
We are using the term 'Filtering' to mean a few different things.  I think we should leave 'filtering' to mean rejecting data that is not used in conditions.  The pendingData is already 'filtered' and now we are ordering/deferring/batching it for submission to the engine.  Perhaps we can change this to PERF__BATCHING_THRESHOLD or something like that, and make analogous naming changes below?